Yoast/wordpress-seo
# Add Spanish verb stem modifications

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* [Yoast SEO Premium] Improves word form recognition for stem-changing verbs in Spanish.

## Relevant technical choices:

* The PR should be tested together with the[ premium-configuration PR](https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/235).

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Test in Premium
* Set your site to Spanish
* Open a post and enter a text of at least 100 words.
* Add `aplicar` as your keyphrase.
* Add the word `apliques` somewhere in the text. 
* Make sure that the keyword density assessment says that 1 instance of the keyphrase is found in the text, and that the eye marker highlights the word `apliques`.
* Change your keyphrase to `dirigir`.
* Enter the word `dirijes` somewhere in the text.
* Make sure that the keyword density assessment says that 1 instance of the keyphrase is found in the text, and that the eye marker highlights the word `dirijes`.

The accompanying [ premium-configuration PR](https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/235) also updates the full forms exception list with a few irregular words. 
* Change your keyphrase to `virgen`.
* Enter the word `virgenes` somewhere in the text.
* Make sure that the keyword density assessment says that 1 instance of the keyphrase is found in the text, and that the eye marker highlights the word `virgenes`.


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Documentation


## Quality assurance


Fixes #https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=99&modal=detail&selectedIssue=LINGO-230
